### DISCUSSION TITLE
Run dnf reinstall using mock command line

### DISCUSSION THREAD

**Discussion by ashuk2000** [rpm-software-management/mock#1105] [Q&A] [answered]
When we use mock --install, it executes dnf install, but I have packages where I have made changes but not made any changes to version.

When I run mock install dnf says packages already installed.
Any way to forceinstall or make mock run dnf reinstall from commandline

**Comment by praiskup** [comment]
There's `dnf reinstall`, a separate command from `dnf install`.  This is not implemented in Mock.

**Comment by praiskup** [comment, marked as answer]
But you can do `mock --dnf-cmd reinstall <your pkg>`.
